fvm/src/engine/mod.rs

```diff
@@ -259,6 +259,16 @@ fn wasmtime_config(ec: &EngineConfig) -> anyhow::Result<wasmtime::Config> {
     // debugging tools. But we'll just have to live with that.
     c.macos_use_mach_ports(false);
 
+    // wasmtime default: true
+    // Disable extended const support. We'll probably enable this in the future but that requires a
+    // FIP.
+    c.wasm_extended_const(false);
```

Comment:
This is new, but all I'm doing here is disabling support for wasm modules that use this feature.